recipes/cython-cmake/recipe.yaml

```diff
@@ -0,0 +1,59 @@
+context:
+  name: cython-cmake
+  version: "0.2.0"
+  username: scikit-build
+  python_min: 3.8
```

Comment:
```suggestion
```
You only should set this if you need 3.10 or newer

Reply:
I think `rattler-build` cannot handle this variant, but I will try it on this PR.

Reply:
It should. cc @wolfv here

Reply:
It does work fine, but there is weird interaction with conda-forge infra. We automatically pick up a `variants.yaml` file, but not a `conda_build_config.yaml`. Staged-recipes does the inverse, it picks up `conda_build_config.yaml` file but ignores the `variants.yaml`.

I need to spend some time next week to figure this out once and for all :)